### DISCUSSION TITLE
Automatic vs. Manual flag to distinguish between automatically extracted features and annotations

### DISCUSSION THREAD

**Discussion by magdalenafuentes** [mir-dataset-loaders/mirdata#453] [Ideas]
Many datasets we're adding have both manually annotated and automatically extracted annotations. Right now, such datasets use the same attributes (e.g. `F0`) indistinctly. One idea is to  add a flag somewhere indicating if the data was manually or automatically extracted. That could be done at the `Dataset` level:
```
source = {"F0": "manual", "beats": "automatic"}
```
Or maybe in the annotation's class in #336 (though this would leave metadata out)

**Comment by rabitt** [comment]
This starts to feel pretty jams-y - since the long term goal is for the annotation data structures to use jams under the hood, I think it's better to put it there.

What's the reasoning for wanting to have this information there in the code now? It's implicitly in the documentation already, right?

**Comment by nkundiushuti** [comment]
In some datasets we have features extracted automatically are labeled as annotations and we are afraid that people may treat this as ground truth. In these cases we want to make it more explicit that these features are automatic, semi-automatic, manually computed. 
I think it should not be enforced but allowed.  
I don't know what's the best way to do it. The lightest way is to have it mentioned in the comments of the loader. The other way is maybe to return it in the info function.

**Comment by rabitt** [comment]
What about introducing optional arguments to the `Annotations` classes, like "source" as you suggest that is stored as part of the object?
